adapter/integration-test/src/adapter-datasource.test.js

```diff
@@ -0,0 +1,211 @@
+const request = require("supertest")
+const waitOn = require("wait-on")
+
+const URL = process.env.ADAPTER_API || "http://localhost:9000/api/adapter";
+
```

Comment:
I put it into and extra file to separate the stateless execution and the management of the configurations

Reply:
I like it. That makes it more maintainable and makes debugging easier